Extension/package.json

```diff
@@ -792,6 +792,11 @@
                 },
                 "default": []
               },
+              "envFile": {
+                "type": "string",
+                "description": "Absolute path to a file containing environment variable definitions. This file has key value pairs sepearted by an equals sign per line. E.g. KEY=VALUE",
```

Comment:
spelling

Reply:
Should we use something like https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker